server/mvc/models/User.js

```diff
@@ -82,7 +82,7 @@ var User = inherit(ThinkyDocumentModel, {
     return (this.document.roles.indexOf("admin") > -1);
   },
 }, {
-  createOrUpdate: function(userId, accessToken, oauthType){
+  createOrUpdate: function(userId, battletag, accessToken, oauthType){
```

Comment:
Should be moved. See: https://github.com/Starbow/website/pull/19/files#r34885331